src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -795,6 +797,7 @@ private void clearAndDisableWinningRuleFields() {
     checkBoxMaxRankingsAllowedMax.setDisable(true);
     textFieldMinimumVoteThreshold.clear();
     textFieldMinimumVoteThreshold.setDisable(true);
+    textFieldStopTabulationEarly.clear();
```

Comment:
Please follow the other examples and `setDisable` too. Then around line 1115, undo it, as is done with the others.